### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by pingpeng1017** [apacha/MusicObjectDetector-TF#12]
Hi, I'm currently working on a program to convert sheet music images into Braille notation, and I came across your Music Object Detector. I was wondering if you could provide access to your pre-trained models, as I believe they could significantly enhance the accuracy of Braille notation conversion. Moreover, I'm also looking to extend the capabilities of the Music Object Detector by adding a feature for Braille notation conversion. Do you think this might be possible? Any insights or advice you can provide on this would be greatly appreciated.

**Comment by apacha** [comment]
Hi, please have a look at https://github.com/apacha/MusicObjectDetector-TF/releases/tag/full-page-detection-v2 

Please keep in mind that this project uses a quite old version of the TensorFlow Object Detection API, which has been expanded and changed since the release of this project. If I were to start an object detection project, I would go with https://github.com/open-mmlab/mmdetection

As for the Braille notation conversion, I don't quite understand what you mean. Do you want to train the detector to be able to detect musical symbols in Braille notation? Or are you trying to convert western music notation into Braille notation?
